### ISSUE TITLE
UnnecessaryStringConcatSniff false positives since upgrade

### ISSUE THREAD

**Issue by ostrolucky** [squizlabs/PHP_CodeSniffer#1940]
False positive since uprade from 9f56110 to 3.2.3 for `PHP_CodeSniffer\Standards\Generic\Sniffs\Strings\UnnecessaryStringConcatSniff.Found` with configuration allowMultiline: true

```php
$this->progressBar->setFormat(
    '<comment>%memory:9s%</comment> [<info>%bar%</info>] '.
    '<comment>%percent:3s%%</comment> %message%'
);
```
